CODEOWNERS

```diff
@@ -60,7 +68,10 @@ catalog/spatial/csw/            @bdeining
 
 catalog/spatial/registry/       @clockard
 
-catalog/ui/catalog-ui-search/   @andrewkfiedler
+catalog/ui/catalog-ui-search/                   @andrewkfiedler
+catalog/ui/catalog-ui-search/pom.xml            @andrewkfiedler @rzwiefel
+catalog/ui/catalog-ui-search/src/**/java/       @rzwiefel
```

Comment:
✏️ Unless you specifically meant to remove `andrewkfielder` from these two lines, you'll need to add him back.

Reply:
I discussed with him already and did mean to remove him from the backend code.